docs/csharp/programming-guide/exceptions/codesnippet/CSharp/using-exceptions_3.cs

```diff
@@ -1,27 +1,32 @@
-        static void TestCatch2()
+        using System;
+        using System.IO;
+
+        public class ExceptionExample
         {
-            System.IO.StreamWriter sw = null;
-            try
+            static void Main()
             {
-                sw = new System.IO.StreamWriter(@"C:\test\test.txt");
-                sw.WriteLine("Hello");
-            }
+                try
+                {
+                    using(var sw = new StreamWriter(@"C:\test\test.txt"))
+                    {
+                    sw.WriteLine("Hello");
```

Comment:
This line should be indented to the right.

Reply:
Thanks for catching that, @jzeferino!